src/olympia/conf/prod/settings.py

```diff
@@ -198,6 +198,17 @@
 if NEWRELIC_ENABLE:
     NEWRELIC_INI = '/etc/newrelic.d/%s.ini' % DOMAIN
 
+FXA_CONFIG = {
+    'client_id': env('FXA_CLIENT_ID'),
+    'client_secret': env('FXA_CLIENT_SECRET'),
+    'content_host': 'https://accounts.firefox.com',
+    'oauth_host': 'https://oauth.accounts.firefox.com/v1',
+    'profile_host': 'https://profile.accounts.firefox.com/v1/profile',
```

Comment:
Actually I think the last `profile` should come off, sorry. The stable URL we use is `https://stable.dev.lcip.org/profile/v1` and it's listed as `https://stable.dev.lcip.org/profile/v1/profile` so I think we still want it to end with `v1`.

Should be `https://profile.accounts.firefox.com/v1`